src/content/events/2024/hackathon-march-2024/sweden-scilifelab.md

```diff
@@ -21,8 +21,27 @@ Contact:
 
 ## Schedule
 
+We are largely sticking to the main schedule displayed [here](https://nf-co.re/events/2024/hackathon-march-2024#schedule). For anyone with access, these are the booked rooms for the event:
+
 - 2024-03-18 - 0900-1700: Alpha 2, Milky Way
 - 2024-03-19 - 0900-1700: Gamma 7, Stockholm
 - 2024-03-20 - 0900-1700: Alpha 2, Milky Way
 
-_Details to be confirmed_
+Like proper Swedes, all other activities like pizza and / or drinks will be discussed on site during the hackathon, so everyone has a say!
+
+## Access
+
+If you do not already have access to the Scilifelab Stockholm building (because for example you work there), you will not be able to access the rooms without assistance. Please be at the Scilifelab reception before 9 am, or, if that is not possible, inform Franziska or Maxime on Slack at what time you will arrive, and she will pick you up.
```

Comment:
```suggestion
::: warning
If you do not already have access to the Scilifelab Stockholm building (because for example you work there), you will not be able to access the rooms without assistance. Please be at the Scilifelab reception before **9 am**, or, if that is not possible, inform [Franziska](https://nfcore.slack.com/team/UGP9YUCKD) or [Maxime](https://nfcore.slack.com/team/UE6D8290F) on Slack at what time you will arrive, and they will pick you up.
:::
```